src/RMKERR.CPP

```diff
@@ -1028,7 +1029,8 @@ LOCAL int presskey( 	// prompt user after error message display
 		erAction == ABT
 		? "Press any key (program will abort) "
 		: "Press 'A' to abort, any other key to continue " );
-	int key = getch();							// input any char
+	int key;
+	std::cin >> key;							// input any char
```

Comment:
Did you test this? I'm pretty sure it is not equivalent behavior, though I don't know if there is a better alternative.